src/decorator/params.ts

```diff
@@ -69,6 +69,32 @@ export function Param(name: string) {
 }
 
 /**
+ * This decorator allows to inject a State object to the controller action parameter.
+ * Applied to class method parameters.
+ *
+ * @param objectName The name of object stored in State
+ */
+export function State(objectName?: string) {
```

Comment:
TBH I didn't understand why this decorator is needed and what its relation to session. Can you please explain?